src/_articles/dart-vm/benchmarking.md

```diff
@@ -2,23 +2,22 @@
 title: Benchmarking the Dart VM
 description: Learn how to benchmark your applications
 original-date: 2012-10-01
-date: 2012-11-01
+date: 2018-07-26
 category: dart-vm
 obsolete: true
 ---
 
 _Written by John McCutchan <br>
-October 2012 (updated November 2012)_
-
-{% comment %}
-update-for-dart-2.0
-{% endcomment %}
-
-<aside class="alert alert-info" markdown="1">
-**Note:**
-This article was initially targeted for the standalone VM under the
-1.x Dart SDK.  It does not reflect the Flutter VM,
-nor does it reflect the changes that are coming in [Dart 2](/dart-2).
+October 2012 (updated November 2012; note updated July 2018)_
+
+<aside class="alert alert-warning" markdown="1">
+  This article was initially targeted for the standalone VM under the
+  1.x Dart SDK.  It does not reflect the Flutter VM,
+  nor does it reflect changes in [Dart 2](/dart-2).
+  For example, [Dartium doesn't work with Dart 2.]({{site.webdev}}/dart-2#tools)
+  Also, checked mode and production mode no longer exist;
+  instead, tools have platform-specific support for expensive development-time
```

Comment:
The assert mechanism is not platform-specific. It just happens that the Flutter framework includes a lot of expensive asserts.

Reply:
I meant to say that it was platform specific how you turn on asserts (and any other platform-specific dev-time stuff). I'll try to clarify.